etest/tests.py

```diff
@@ -33,7 +33,8 @@ def __init__(self, ebuild, with_test_phase=False, base_docker_image="alunduil/et
         self.time = datetime.timedelta(0)
         self.output = ""
 
-        self.base_docker_image = base_docker_image
+        self.arch = arch
+        self.base_docker_image = f"alunduil/etest:{arch}"
```

Comment:
Why not leave this as it was and just ensure there is a constructor for the etest image based on arch?  That's more extensible in case someone makes their own image for their own purposes.

Reply:
This came in handy on the next commit, since sometimes the names used between the docker images and the gentoo keywords mismatch. However, we should figure out a cleaner way to do it.

Reply:
Yeah, that's why I was suggesting a way to generate a base docker image from an arch (just a constructor; possibly a newtype).  That makes the caller responsible for this detail and leaves this interface more general.

Reply:
You can see the thing that Ive mentioned on the following commit. I think we can move the ppc check from there to the build script, but I cant think of an elegant way to differentiate between armv5 and armv7 without that "hack"